lib/cluster.js

```diff
@@ -107,6 +108,19 @@ Cluster.prototype.getSeedList = function getSeedList(callback) {
     });
 };
 
+Cluster.prototype.getConnectionErrorCount = function getConnectionErrorCount() {
+    return this.connectionErrorCount;
+};
+
+Cluster.prototype.printConnectionErrorMsg = function printConnectionErrorMsg() {
+    var count = this.getConnectionErrorCount();
+    if (count > 0) {
+        console.error('Failed to connect to %d nodes.', count);
+    } else {
+        console.error('Successfully connected to all nodes.');
```

Comment:
I'd suggest that if there are no problems then you don't need to print anything, unless you're in some kind of verbose mode.